.github/actions/combine-multi-arch-images/action.yaml

```diff
@@ -54,55 +54,98 @@ runs:
     - name: Get base tags
       id: get-base-tags
       run: |
-        amd64_tags=$(printf "%s\n" $ALL_TAGS | grep "\-amd64" | sed "s/-amd64$//g")
-        arm64_tags=$(printf "%s\n" $ALL_TAGS | grep "\-arm64" | sed "s/-arm64$//g")
-        base_tags=$(printf "%s\n" "$amd64_tags" "$arm64_tags" | sort | uniq)
+        echo "All tags: ${{ steps.get-all-tags.outputs.tags }}"
```

Comment:
I think it would be redundant.
https://github.com/autowarefoundation/autoware/actions/runs/11219103356/job/31184311855#step:3:576
```suggestion
```